doc/tools/make_rst.py

```diff
@@ -61,22 +59,22 @@
     "This method doesn't need an instance to be called, so it can be called directly using the class name.",
     "This method describes a valid operator to use with this type as left-hand operand.",
 ]
-strings_l10n = {}
+strings_l10n: Dict[str, str] = {}
 
-STYLES = {}
+STYLES: Dict[str, str] = {}
 
 
-def print_error(error, state):  # type: (str, State) -> None
+def print_error(error: str, state: "State") -> None:
```

Comment:
Is this one a string because `State` isn't in scope yet? If so, should this just be defined after `State` so the ref can be used?

Reply:
Yes & can move it.